### ISSUE TITLE
Delete pipeline executions when pipeline is deleted

### ISSUE THREAD

**Issue by jindrichmynarz** [linkedpipes/etl#386]
When a pipeline is deleted, its executions remains available. However, when you go to the debug view of an execution of a deleted pipeline, "Can't load data: Not found" alert is displayed. in order to avoid this issue, pipeline executions should be deleted when a pipeline is deleted.

Tested using the develop version f73314dc3ef7f8ce67a99bda1a88a2eb4c84d98f.

**Comment by jakubklimek** [comment]
This is a known behavior. We are still not in the phase where we could affort do spend time on preventing users to do something stupid. We are still focusing on enabling users to do whats reasonable. Nevertheless, it is ok to have this documented

**Comment by jindrichmynarz** [comment]
I don't think that it is preventing users to do stupid things, I think it's a convenience. When a pipeline is deleted, as a convenience, its executions are deleted too.

**Comment by jakubklimek** [comment]
This can be done better. A copy of the actual executed pipeline is stored with the execution and can be used for viewing it even though the original pipeline is changed or deleted. This would be the proper behavior, however, it is not used this way now.

On the other hand it is not that unexpected that when you delete something that is in use, it breaks the things that use it.

To sum up, yes, something could be done about this, just not sure when.
